morris/roster.json

```diff
@@ -39,5 +39,9 @@
 		{
 			"name": "Maximilian Hartley"
 		}
+
```

Comment:
We'll need to make sure there's a comma in-between each object! Please could you add one between Max's and yours? :)